modules/ctconfiguration/ctconfiguration.php

```diff
@@ -0,0 +1,197 @@
+<?php
+
+/**
+ * Class CTConfiguration
+ *
+ * @property $bootstrap
+ */
+class CTConfiguration extends Module
+{
+    /**
+     * CTConfiguration constructor.
+     */
+    public function __construct()
+    {
+        $this->name    = 'ctconfiguration';
+        $this->tab     = 'front_office_features';
+        $this->version = '1.0.0';
+        $this->author  = 'Community Theme';
+
+        parent::__construct();
+
+        $this->displayName = $this->l('Community Configuration');
+        $this->description = $this->l('Configuration for community theme blocks and content.');
+        $this->ps_versions_compliancy = array('min' => '1.6', 'max' => '1.6.1.4');
+
+        $this->bootstrap = true;
+    }
+
+    /**
+     * Installs module to PrestaShop
+     *
+     * @return bool
+     */
+    public function install()
+    {
+        parent::install();
+
+        epr('Install ctconfiguration');
+
+        $hooksToUnhook = array(
+            array('module' => 'blockcategories', 'hook' => 'footer',)
+        );
+        foreach ($hooksToUnhook as $unhook) {
+            $this->unhookModule($unhook['module'], $unhook['hook']);
+        }
+
+        $hooksToInstall = array('displayHeader');
+        foreach ($hooksToInstall as $hookName) {
+            $this->registerHook($hookName);
```

Comment:
well, theme will work fine with 1.6.1.4+, right? so we can use 1.6.1.4 features only